vault/seal.go

```diff
@@ -272,6 +298,19 @@ type SealConfig struct {
 
 	// How many keys to store, for seals that support storage.
 	StoredShares int `json:"stored_shares"`
+
+	// VerificationRequired indicates that after a rekey validation must be
+	// performed (via providing shares from the new key) before the new key is
+	// actually installed. This is omitted from JSON as we don't persist the
+	// new key, it lives only in memory.
+	VerificationRequired bool `json:"-"`
+
+	// VerificationKey is the new key that we will roll to after successful
+	// validation
+	VerificationKey []byte `json:"-"`
```

Comment:
Maybe the keys in process could live in this object instead of on core?

Reply:
I think that's a fine refactor for later, but that would be something affecting not just the verification logic but the existing rekey logic.

Reply:
Updated.